segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/StreamSegmentService.java

```diff
@@ -119,6 +119,15 @@ public StreamSegmentService(SegmentContainerRegistry segmentContainerRegistry, S
                 "mergeTransaction", targetStreamSegment, sourceStreamSegment);
     }
 
+    @Override
+    public CompletableFuture<MergeStreamSegmentResult> mergeStreamSegment(String targetStreamSegment, String sourceStreamSegment,
+                                                                          Collection<AttributeUpdate> attributes, Duration timeout) {
+        return invoke(
+                sourceStreamSegment,
+                container -> container.mergeStreamSegment(targetStreamSegment, sourceStreamSegment, attributes, timeout),
+                "mergeTransaction", targetStreamSegment, sourceStreamSegment);
```

Comment:
`mergeStreamSegment`

Please fix above too - I believe this has been inconsistent for a very long time.

Reply:
Done.